integrations/postgresql/src/test/kotlin/xef/PGVectorStoreSpec.kt

```diff
@@ -23,12 +29,12 @@ val postgres: PostgreSQLContainer<Nothing> =
 
 class PGVectorStoreSpec :
   StringSpec({
-    val container = install(SharedTestContainerExtension(postgres))
+    val container = install(ContainerExtension(postgres))
     val dataSource =
       autoClose(
         HikariDataSource(
           HikariConfig().apply {
-            jdbcUrl = container.jdbcUrl
+            jdbcUrl = container.jdbcUrl.replace("localhost", "0.0.0.0") // workaround: needs to be improved
```

Comment:
I had trouble executing the tests: they couldn't connect to the database because using `localhost` as the host causes that the container cannot be found. This should be solved somehow else, let me know any ideas!

Reply:
Maybe just use 0.0.0.0 in the test. We should not handle address resolution problems ourselves.

Reply:
Then I leave it like that!